registry/remote/repository.go

```diff
@@ -320,10 +320,12 @@ func (r *Repository) tags(ctx context.Context, fn func(tags []string) error, url
 	return parseLink(resp)
 }
 
-// UpEdges returns the manifest descriptors directly referencing the given
+// Predecessors returns the manifest descriptors directly referencing the given
 // manifest descriptor.
+// Predecessors internally leverages Referrers, and converts the result artifact
+// descriptors to OCI descriptors.
```

Comment:
You may want to state that this function only works on manifests.

Reply:
line #324 mentions this.